docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -0,0 +1,205 @@
+Model Builder is a graphical Visual Studio extension. 
+
+![Model Builder Visual Studio extension user interface animation](media/ml-dotnet-model-builder.gif)
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Scenarios
+
+You can bring many different scenarios to Model Builder, to generate a machine learning model for your application.
+
+A scenario is a description of the type of prediction you want to make on your data. For example
+- predict future product sales volume based on historical sales data
+- classify sentiments as positive or negative based on customer reviews
+- detect whether a banking transaction is fraudulent
+- route customer feedback issues to the correct team in your company
+
+In Model Builder, you need to map your scenario onto an [ML.NET task](resources/tasks.md). You can use Model Builder for **regression** (predicting numbers) and **classification** (predicting categories).
+
+### Which machine learning scenario is right for me?
+
+Model Builder comes with templates for sentiment analysis, issue classification, price prediction, and custom scenarios. These templates can be used as a starting point for your specific ML.NET scenario.
+
+#### Sentiment analysis (binary classification)
+
+Sentiment analysis can be used to predict positive or negative sentiment of customer feedback. It is an example of the binary classification task.
+
+Binary classification is used to categorize data into two classes (yes/no; pass/fail; true/false; positive/negative). It can be used to answer questions such as:
+
+- Is this email spam? (spam detection)
+- Which applicants may be eligible for membership? (application screening)
+- Which accounts may not pay their invoices on time? (risk mitigation)
+- Is this credit card transaction fraudulent? (fraud detection)
+
+If your scenario requires classification into two categories, you can use this template with your own dataset.
+ 
+#### Issue classification (multiclass classification)
+
+Issue classification can be used to categorize customer feedback (for example, on GitHub) issues using the issue title and description. It is an example of the multi-class classification task.
+
+Multiclass classification can be used to categorize data into three or more classes. It can be used to answer questions such as:
+
+- To which department should I route a support ticket? (support ticket routing)
+- What is the priority of a customer issue? (customer issue prioritization)
+- What category does a product belong to? (product classification)
+- What type of document? (document/email classification)
+
+You can use the issue classification template for your scenario if you want to categorize data into three or more categories.
+
+#### Price prediction (regression)
+
+Price prediction can be used to predict house prices using location, size, and other characteristics of the house. It is an example of the regression task.
+
+Price prediction can be used to predict house prices using location, size, 
```

Comment:
```suggestion
Price prediction can be used to predict house prices using location and size.
```

Reply:
That was actually an accidental repetition. Well spotted